ts/Dashboards/Components/EditableOptions.ts

```diff
@@ -98,5 +57,93 @@ class EditableOptions {
         return this.component.options.editableOptions;
     }
 }
+namespace EditableOptions {
+
+    /**
+     * Configuration for a single option in editable options. If type is
+     * `nested` the options are rendered in the accordion menu, with rest of the
+     * options defined in the detailed options.
+     */
+    export interface Configuration {
```

Comment:
Any suggestions for the new name of the `Configuration` interface?

Reply:
> The EditableOptions interface which is in the Component.ts namespace is a bit missleading, or even unnecessary. 

Could you please check it? If it's unnecessary, it should be removed.

> We could move all the contents of this interface to the ComponentOptions interface. IMO, we could delete it from component and leave only the EditableOptions class. (I think it is also a bit redundant, but since it was there in the original code, I left it, when refactoring the Sidebar).

The first part sounds good, the second part confuses me a little, sorry. Do you mean to move the content to ComponentOptions and then delete it?

I know it might not be part of this task, but naming is a bit confusing here. My first thought was `EditableOptions` doesn't sound like the correct name for the Class. Then we could go with simple `NewClassName.Options`, just like in the other Classes